pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -206,20 +185,19 @@ func (am *Alertmanager) StopAndWait() error {
 	close(am.stopc)
```

Comment:
I know this has changed but should we make sure that this doesn't panic? We could make this re-entrant?

Reply:
Any suggestions? I think it's a bit of low risk. The race can only occur if: we remove an org, then we're about to remove that org's alertmanager just as we're about to shut down Grafana.

Reply:
Discussed offline with George, agreed on not taking any action right now.